CHANGELOG.md

```diff
@@ -80,6 +80,7 @@ Note that this project **does not** adhere to [Semantic Versioning](http://semve
 - We fixed an issue where the name fields were not updated after the preferences changed. [#6515](https://github.com/JabRef/jabref/issues/6515)
 - We fixed an issue where "null" appeared in generated BibTeX keys. [#6459](https://github.com/JabRef/jabref/issues/6459)
 - We fixed an issue where the authors' names were incorrectly displayed in the authors' column when they were bracketed. [#6465](https://github.com/JabRef/jabref/issues/6465) [#6459](https://github.com/JabRef/jabref/issues/6459)
+- We fixed an issue where the `Show names unchanged` option did not convert LaTeX macros into Unicode. [#6479](https://github.com/JabRef/jabref/pull/6479)
```

Comment:
Just a note: 
You don't need to add a changelog entry, if the error was introduced in a dev version.
Changelog should show differences between actual release and (current dev/next release version)

Reply:
Aha! I try to follow the [guidelines] but it is not always clear to me =/

Reply:
No problem  ;)  If you encounter any inconsistencies just give us a hint.  We are happy for every contributor

Reply:
So far no inconsistency 😛 
I appreciate that ending up attempting to fix your own PR post-merge should be an exception rather than the rule 🤡

Reply:
I mean, if I end up thinking that someone else would benefit from the information I will submit a PR to the documentation

Reply:
In an ideal world it should never happen 😇 ,  but we are still all humans (at least I hope so 🤖 ), so bugs are inevitable. And fixing things after a merge happens to everyone, so welcome to the club ;)